.github/workflows/build-and-test-all.yml

```diff
@@ -70,6 +70,7 @@ jobs:
       - run: inv generate-coverage-info ./testrunner $GITHUB_WORKSPACE
```

Comment:
For `recursor` and `dnsdist` jobs, this step is also skipped. Should we also add here `if: ${{ env.COVERAGE == 'yes' }}`?